recipes/ois/all/conanfile.py

```diff
@@ -1,17 +1,25 @@
 import os
 import glob
-from conans import ConanFile, tools, CMake
-from conans.errors import ConanInvalidConfiguration
+from conan import ConanFile
+from conan.errors import ConanInvalidConfiguration
+from conan.tools.microsoft import check_min_vs, is_msvc_static_runtime, is_msvc
+from conan.tools.files import apply_conandata_patches, export_conandata_patches, get, copy, rm, rmdir, collect_libs
+from conan.tools.build import check_min_cppstd
+from conan.tools.scm import Version
+from conan.tools.cmake import CMake, CMakeToolchain, cmake_layout
+from conan.tools.env import VirtualBuildEnv
+
+required_conan_version = ">=1.53.0"
+
 
 class OisConan(ConanFile):
```

Comment:
Upstream defines CXX version of 11 https://github.com/wgois/OIS/blob/master/CMakeLists.txt#L6, so you'll want to add a `validate()` method to check it. [See this for an example on how to implement it](https://github.com/conan-io/conan-center-index/blob/master/docs/package_templates/cmake_package/all/conanfile.py#L78)

Reply:
Done